react/features/app/actions.web.js

```diff
@@ -93,7 +94,7 @@ export function appNavigate(uri: ?string) {
         let url = `${baseURL}config.js`;
 
         // XXX In order to support multiple shards, tell the room to the deployment.
-        room && (url += `?room=${getBackendSafeRoomName(room)}`);
+        room && (url = appendURLParam(url, 'room', getBackendSafeRoomName(room)));
```

Comment:
I think it's a noop, by why didn't you add the release here?

Reply:
cause its noop :) but I'll add it to match the native code